docs/source/notebooks/tpt.ipynb

```diff
@@ -0,0 +1,779 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Transition path theory\n",
+    "\n",
+    "Transition path theory (TPT) is a method to study the ensemble of reactive trajectories, i.e., trajectories which come from a defined set of states $A$ and go next to $B$. It can answer at which rate they occur, as well as depict parallel pathways, traps, sequences of events, etc. Furthermore it introduces the notion of 'committor functions', which deals with probabilities of ending up in set $A$ or $B$ given the trajectory starts at some state potentially outside $A\\cup B$.\n",
+    "\n",
+    "A mathematical description of TPT can be found in <cite data-cite=\"nbtpt-weinan2006towards\">(Weinan, 2006) and </cite><cite data-cite=\"nbtpt-metzner2009transition\">(Metzner, 2009)</cite>. The implementation is based on <cite data-cite=\"nbtpt-noe2009constructing\">(Noe, 2009)</cite>. Coarse-graining by path decomposition is presented in <cite data-cite=\"nbtpt-noe2009constructing\">(Noe, 2009)</cite> and <cite data-cite=\"nbtpt-berezhkovskii2009reactive\">(Berezhokovskii, 2009)</cite>.\n",
+    "    \n",
+    "To demonstrate the TPT API ([API docs here](../api/generated/sktime.markov.ReactiveFlux.rst#sktime.markov.ReactiveFlux)), in the following the example of a drunkard's walk is presented. The example is motivated by <cite data-cite=\"nbtpt-doyle1984random\">(Doyle, 1984)</cite> and <cite data-cite=\"nbtpt-valleriani2015circular\">(Valleriani, 2015)</cite>, where a drunkard is placed on a network of states and two special states, home and the bar. When the drunkard reaches either of these special states the trajectory stays there with high probability.\n",
+    "One can then ask which paths can be taken and also with which probability the drunkard is going to reach either of the states given a certain current position."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "To this end, import sktime and numpy for general numerical operations."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import numpy as np\n",
+    "import sktime"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We can create a [DrunkardsWalk](../api/generated/sktime.data.drunkards_walk.rst#sktime.data.drunkards_walk) simulator by specifying bar and home locations. As the drunkard lives on a 2-dimensional surface, the locations are given in terms of integer coordinates. Internally, this is related back to $\\mathrm{width}\\times\\mathrm{height}$ states. On a 'normal' state (i.e., not home/bar or barrier), the drunkard can stagger a step within a $3\\times 3$ grid with uniform probability without staying on the same spot."
```

Comment:
this still doesn't totally make sense to me (the figures do, but the explanation is hard, although it's probably a me problem)

Reply:
changed the explanation so that it hopefully is clearer

Reply:
thank you!